docs/manual/determinism-checker.tex

```diff
@@ -378,7 +378,43 @@
 E set(@PolyDet List<E> this, @PolyDet("use") int index, E element)
 \end{Verbatim}
 
+Whenever \<@PolyDet("up")>, \<@PolyDet("down")>, and \<@PolyDet("upDet")> are written on formal parameters,
```

Comment:
So, does this mean you are going to close https://github.com/t-rasmud/checker-framework/pull/127?